extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -39,6 +39,7 @@ object KinesisUtils {
    * @param ssc    StreamingContext object
    * @param streamName   Kinesis stream name
    * @param endpointUrl  Url of Kinesis service (e.g., https://kinesis.us-east-1.amazonaws.com)
+   * @param regionName   Region name to indicate the location of the Amazon Kinesis service
```

Comment:
same here...

"The Amazon DynamoDB table and Amazon CloudWatch metrics associated with your application will also use this region setting."